testsuite/features/secondary/minssh_tunnel.feature

```diff
@@ -61,7 +61,7 @@ Feature: Register a Salt system to be managed via SSH tunnel
     And I follow "Software" in the content area
     And I follow "List / Remove"
     And I enter "milkyway-dummy" as the filtered package name
-    And I click on the filter button
+    And I click on the filter button until page does contain "milkyway-dummy" text
```

Comment:
That UX looks like a weird mechanism to me.
We will be clicking on the filter button multiple times until the page does not contain the package?